tests/test_order_registry.py

```
@@ -0,0 +1,1186 @@
+
+
+def test_size_matched_float_accumulation_needs_epsilon(registry: OrderRegistry):
+    """Forty partial fills of 2.5 sum to 100.0 only within SIZE_EPS."""
+    local_id = str(uuid.uuid4())
+    now_ms = 1723840000000
+    registry.create_order(
+        OrderRecord(
+            id=local_id,
+            order_id="0xvenue_order_eps",
+            condition_id="0xcond_eps",
+            token_id="0xtok_eps",
```

Ruff (0.16.2)

[error] Possible hardcoded password assigned to argument: "token_id"

(S106)